manual/manual/refman/exten.etex

```diff
@@ -1815,52 +1828,6 @@ let y = [%extension_constructor Y]
  x <> y;;
 \end{caml_example}
 
-\section{Quoted strings}\label{s:quoted-strings}
-
-(Introduced in OCaml 4.02)
-
-Quoted strings "{foo|...|foo}" provide a different lexical syntax to
-write string literals in OCaml code. They are useful to represent
-strings of arbitrary content without escaping -- as long as the
-delimiter you chose (here "|foo}") does not occur in the string
-itself.
-
-\begin{syntax}
-string-literal: ...
-     |  '{' quoted-string-id '|'  ........ '|' quoted-string-id '}'
-;
-quoted-string-id:
-     { 'a'...'z' || '_' }
-;
-\end{syntax}
-
-The opening delimiter has the form "{id|" where "id" is a (possibly
-empty) sequence of lowercase letters and underscores.  The
-corresponding closing delimiter is "|id}" (with the same
-identifier). Unlike regular OCaml string literals, quoted
-strings do not interpret any character in a special way.
-
-Example:
-
-\begin{verbatim}
-String.length {|\"|}         (* returns 2 *)
-String.length {foo|\"|foo}   (* returns 2 *)
-\end{verbatim}
```

Comment:
Note that we're losing this example, perhaps move it to the tutorial? It's useful.

Reply:
I would propose to add to the tutorial example
```OCaml
{|inside a quoted string, neither \ nor " are special characters|};;
  {delimiter|the end of this  |} quoted string is here|delimiter}
= "the end of this |} quoted string is here";;
```
to solve both problems at the same time.

Reply:
I like, but the dumb example `String.length {|\"|}         (* returns 2 *)` does remain useful and might go _somewhere_. It explains to beginners why you would want such a thing at all, which is to avoid leaning toothpick syndrome.

Reply:
I missed your point then. I would say that for this case 
```OCaml
"\"\\\\\""={|"\\"|}
```
is a better example.

Reply:
That would do it too.